src/components/navigation/header/header.module.css

```diff
@@ -149,3 +145,43 @@
   composes: button;
   background: url("/_assets/menu-close.svg") no-repeat 50% 50%;
 }
+
+.breadCrumbMenu {
+  position: relative;
```

Comment:
Is this `position` needed?

Reply:
I think with the current jfx structure, we don’t need it. I’ll check and remove it if that’s the case.